language/stdlib/transaction_scripts/publish_shared_ed25519_public_key.move

```diff
@@ -32,7 +32,15 @@ fun publish_shared_ed25519_public_key(account: &signer, public_key: vector<u8>)
     SharedEd25519PublicKey::publish(account, public_key)
 }
 spec fun publish_shared_ed25519_public_key {
+    use 0x1::Errors;
+
     include SharedEd25519PublicKey::PublishAbortsIf{key: public_key};
     include SharedEd25519PublicKey::PublishEnsures{key: public_key};
+
+    aborts_with [check]
+        Errors::INVALID_STATE,
+        Errors::ALREADY_PUBLISHED,
+        Errors::INVALID_ARGUMENT,
+        Errors::NOT_PUBLISHED; // TODO: Prover cannot prove `account` has LibraAccount published.
```

Comment:
I think this is fine to leave out since the existence of `account` is being checked in the script prologue.

Reply:
Removed this error code by adding a "requires" clause.